### PR TITLE
Review product type integration test refactor

### PR DESCRIPTION
**Pull request by barbara79**
1. code and add tests that cover the created code. Your code should be warning-free.
   * [ ] tests existing
1. stick to PSR-2 and and don't reformat existing code.
   * [ ] code style check succesful

### CODE REVIEW

File: tests/integration/Channel/ChannelFixture.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+use Commercetools\Core\Client\ApiClient;
+use Commercetools\Core\Helper\Uuid;
+use Commercetools\Core\IntegrationTests\ResourceFixture;
+use Commercetools\Core\Model\Channel\Channel;
+use Commercetools\Core\Model\Channel\ChannelDraft;
+use Commercetools\Core\Request\Channels\ChannelCreateRequest;
+use Commercetools\Core\Request\Channels\ChannelDeleteRequest;
+
+class ChannelFixture extends ResourceFixture
```
Comment (barbara79) [inline tests/integration/Channel/ChannelFixture.php]: this class is in the PR #523

File: tests/integration/State/StateFixture.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+use Commercetools\Core\Client\ApiClient;
+use Commercetools\Core\Helper\Uuid;
+use Commercetools\Core\IntegrationTests\ResourceFixture;
+use Commercetools\Core\Model\State\State;
+use Commercetools\Core\Model\State\StateDraft;
+use Commercetools\Core\Request\States\StateCreateRequest;
+use Commercetools\Core\Request\States\StateDeleteRequest;
+
+class StateFixture extends ResourceFixture
```
Comment (barbara79) [inline tests/integration/State/StateFixture.php]: this Pr is in #528

File: tests/integration/Type/TypeFixture.php [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+use Commercetools\Core\Model\Common\LocalizedString;
+use Commercetools\Core\Model\Type\FieldDefinition;
+use Commercetools\Core\Model\Type\FieldDefinitionCollection;
+use Commercetools\Core\Model\Type\StringType;
+use Commercetools\Core\Model\Type\Type;
+use Commercetools\Core\Model\Type\TypeDraft;
+use Commercetools\Core\Request\Types\TypeCreateRequest;
+use Commercetools\Core\Request\Types\TypeDeleteRequest;
+
+class TypeFixture extends ResourceFixture
```
Comment (barbara79) [inline tests/integration/Type/TypeFixture.php]: this class is in PR https://github.com/commercetools/commercetools-php-sdk/pull/524

File: tests/integration/ProductType/ProductTypeUpdateRequestTest.php [hunk 4]
```diff
@@ -393,479 +388,82 @@ public function testReferenceAttributeDefinition()
...
+                    'new-test',
+                    $result->getAttributes()->current()->getType()->getValues()->current()->getLabel()->en
+                );
+
+                return $result;
+            }
         );
     }
-
-    public function testChangeSearchable()
```
Comment (jenschude) [inline tests/integration/ProductType/ProductTypeUpdateRequestTest.php]: Some of the following tests seem to be missing

**Pull request merged**